### ISSUE TITLE
Getting set up: Object.entries is not a function

### ISSUE THREAD

**Issue by 1j01** [jadjoubran/webdash#67]
```
+ webdash@1.2.0
added 67 packages from 58 contributors and updated 1 package in 103.9s
```
```
$ npx webdash
Object.values is not a function
```
```
$ npx webdash serve
Object.entries is not a function
```

**Comment by jadjoubran** [comment]
Thanks for trying webdash!

I think you're running a version of node that is older that 8.0
Can you please update node and try again?

**Comment by 1j01** [comment]
Theoretically there would have been a warning about `engines` not matching, but it would be nice if it checked the version of Node within the code, or features like this that it needs. Or if it was mentioned on the docs what version of Node you need. Oh wait, it does - did you just add that? Nah, it was probably already there. It would be nice if it was a *step*, highlighted as something you need to check.
(I'm pretty sure I pasted in that whole thing after a quick glance of it 😳)

**Comment by jadjoubran** [comment]
I'm not sure if `engines` made it to the last release or not, but it'll be available in 1.3
but anyway it's just a warning on installation, I feel like people tend to skip these warnings
so in v1.3 there'll be a warning when you run webdash serve, as shown here

<img width="624" alt="38050840-0be84728-32d5-11e8-8566-813925d19414" src="https://user-images.githubusercontent.com/2265232/38171084-f3bb5c12-359a-11e8-9882-1e33e0ed35f6.png">

reference: #46

**Comment by 1j01** [comment]
Sweet!
